src/site/xdoc/downloads.xml

```diff
@@ -113,23 +113,23 @@ under the License.
     </tr>
     <tr>
       <td style="test-align: left">
-        1.4.12
+        1.4.13
       </td>
       <td style="test-align: left">
-        2019/11/29
+        2020/02/29
       </td>
       <td style="test-align: left">
-        <a href="https://apache.org/dist/hbase/hbase-1.4.12/compat-check-report.html">1.4.11 vs 1.4.12</a>
+        <a href="https://apache.org/dist/hbase/1.4.13/compat-check-report.html">1.4.12 vs 1.4.13</a>
```

Comment:
did a release script choose this directory name? I want to get us consistently naming the main repo releases as "hbase-X.Y.Z".

Reply:
oops! my bad @busbey . Let me keep this in mind from next time, because I didn't use the script for this one.